test/mlir/onnx/onnx_check_verbosity.mlir

```diff
@@ -0,0 +1,6 @@
+// REQUIRES: system-linux
```

Comment:
Is there a reason this can't run on Windows?

Reply:
I can try to add the support. I do not have a Windows system to try on so might take a couple of runs on the CI to get it right (painful given it still takes too long to run CI on Win).

Reply:
I see that you split the test into a Windows version and a Unix version. Does it make sense to update the checks to work for both (even though we'd lost some flags)? Alternatively, can we use separate check labels for Windows/Unix? I forget if that works.

Reply:
I don't think it makes sense to update the checks so that they work across Windows/Linux because there are too many differences, even the command names are different between the architectures. 
Still working on the test, I do not know how to use difference CHECK-LABELS if the CHECK depends on the system architecture.